components/org.wso2.carbon.ldap.server/src/main/resources/embedded-ldap.xml

```diff
@@ -48,8 +48,8 @@
   <!-- Default partition admin configurations -->
   <PartitionAdmin>
     <Property name="uid">admin</Property>
-    <Property name="firstName">admin</Property>
-    <Property name="lastName">admin</Property>
+    <Property name="firstName">John</Property>
```

Comment:
Hey..  instead "John Doe" shall we use something like "System Administrator". Something gender neutral. WDYT?

Reply:
It would look like following,
<img width="1053" alt="Screen Shot 2019-10-27 at 7 14 56 PM" src="https://user-images.githubusercontent.com/1617810/67635500-1c2b1080-f8ee-11e9-9621-dedff8804923.png">

But i'm -1 to have firstname as "System" and lastname as "Administrator" since,
* Those values are too mechanic to me :)
* Those values will be used in other places\samples. So even we show those two together in here, for other service providers or even in the profile section in the user-portal itself would lead to containing not suitable values.

Ex:
<img width="837" alt="Screen Shot 2019-10-27 at 7 17 48 PM" src="https://user-images.githubusercontent.com/1617810/67635568-c440d980-f8ee-11e9-86a0-1e40f8d8c9f0.png">

Yet, I agree if we can have gender neutral name here, it would be better.

Reply:
Shall we not have First-name & Last-name for super admin by default and map it to fallback - So it could be just "Admin" and title will say "Welcome, Admin", until user change it to something. WDYT?